src/edu/washington/escience/myria/expression/evaluate/GenericEvaluator.java

```diff
@@ -121,27 +131,93 @@ public String getJavaExpressionWithAppend() {
   }
 
   /**
+   * Holder class for results and result counts from {@link #evaluateColumn}.
+   */
+  public static class EvaluatorResult {
+    private final ReadableColumn results;
+    private final Column<?> resultCounts;
+
+    protected EvaluatorResult(
+        @Nonnull final ReadableColumn results, @Nonnull final Column<?> resultCounts) {
+      this.results = results;
+      this.resultCounts = resultCounts;
+    }
+    /**
+     * @return a {@link ReadableColumn} containing results from {@link #evaluateColumn}
+     */
+    public ReadableColumn getResults() {
```

Comment:
Seems this method is not needed because we can also pass `List<Column<?>>` to `outputBuffer.appendFromColumn()` in `Apply`.